planner_node:
  birrts:
    rewire_time_range: 10.0
    # steer_time: 0.1
    steer_time: 1.0
    # max_time: 0.1
    max_time: 5.0
    check_colli_res: 0.01